Formula/solr@8.11.rb

```diff
@@ -0,0 +1,58 @@
+class SolrAT811 < Formula
+  desc "Enterprise search platform from the Apache Lucene project"
+  homepage "https://solr.apache.org/"
+  url "https://dlcdn.apache.org/lucene/solr/8.11.2/solr-8.11.2.tgz"
+  mirror "https://archive.apache.org/dist/lucene/solr/8.11.2/solr-8.11.2.tgz"
+  sha256 "54d6ebd392942f0798a60d50a910e26794b2c344ee97c2d9b50e678a7066d3a6"
+  license "Apache-2.0"
+
+  keg_only :versioned_formula
+
+  depends_on "openjdk"
+
+  def install
+    pkgshare.install "bin/solr.in.sh"
+    (var/"lib/solr").install "server/solr/README.txt", "server/solr/solr.xml", "server/solr/zoo.cfg"
```

Comment:
Should we really install files into `var`? That's not in `HOMEBREW_CELLAR`.

Reply:
Open to changing it, although the [current `solr` formula does it too](https://github.com/Homebrew/homebrew-core/blob/64930eaad53c966bc50eb0b151c02f11644b16f2/Formula/solr.rb#L17), as does the [older `solr@7.7` formula](https://github.com/Homebrew/homebrew-core/blob/64930eaad53c966bc50eb0b151c02f11644b16f2/Formula/solr@7.7.rb#L24).

Reply:
The latter 2 look like config files, so I think `var` was used to retain modifications. Not sure if there is any risk of someone installing both `solr@8.11` and `solr` simultaneously as we use the same directories for pid, logs, and configs (though it may make it simpler for someone who was using `solr` to downgrade back to `solr@8.11`).

Reply:
> The latter 2 look like config files, so I think var was used to retain modifications. 

`etc` retains modifications and is most appropriate for config files.

Reply:
The directory is also `SOLR_HOME` which defaults to `/var/solr/data` for upstream's production installation. It contains Lucene index files.

Ref: https://solr.apache.org/guide/8_11/taking-solr-to-production.html#solr-home-directory